streams-contrib/streams-persist-elasticsearch/src/main/java/org/apache/streams/elasticsearch/ElasticsearchPersistWriter.java

```diff
@@ -139,18 +141,8 @@ public void write(StreamsDatum streamsDatum) {
 
         checkForBackOff();
 
-        LOGGER.debug("Write Document: {}", streamsDatum.getDocument());
-
-        Map<String, Object> metadata = streamsDatum.getMetadata();
-
-        LOGGER.debug("Write Metadata: {}", metadata);
-
-        String index = ElasticsearchMetadataUtil.getIndex(metadata, config);
```

Comment:
What is your justification on removing this? It could be used in downstream implementations